src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useContext } from 'react'
+import './Navigation.css'
+import activeModalContext from '../../utils/activeModal'
+
+function Navigation() {
+
+  const {activeModal, setActiveModal} = React.useContext(activeModalContext);
+
+  const handleAbout = () => {
+    setActiveModal("about");
+  }
+  return (
+    <>
+      <div className='navigation'>
+        <ul className='navigation__list'>
+            <li className='navigation__item'>LOGIN TO STEAM</li>
+            <li className='navigation__item' onClick={() => handleAbout()}>ABOUT</li>
```

Comment:
Please, use `Link` to redirect to the `About` page rather than open a `modal`. It's **navigation** rather than just buttons for opening something